rules/configuration.smk

```diff
@@ -367,7 +371,8 @@ def convert_size(size_bytes, order=None):
 
 
 # by default only one download in parallel (workflow fails on multiple on a single node)
-workflow.global_resources.update({'parallel_downloads': 1, 'deeptools_limit': 1, 'R_scripts': 1})
+workflow.global_resources = {**{'parallel_downloads': 1, 'deeptools_limit': 1, 'R_scripts': 1},
+                             **workflow.global_resources}
```

Comment:
Now we do not overwrite what the user supplies!

Reply:
oh nice!